examples/capable/src/capable.rs

```diff
@@ -0,0 +1 @@
+include!(concat!(env!("OUT_DIR"), "/skel.rs"));
```

Comment:
instead of having to create a new file for this, does having this line inside the src/main.rs file work?

Reply:
Done.